rasa/core/constants.py

```diff
@@ -1,3 +1,14 @@
+import sys
+
+try:
+    # sys.argv is a list so if we find "--request-timeout"
+    # the next item on the list is the passed value
+    arg_index = sys.argv.index("--request-timeout")
```

Comment:
Is there a better way to set this constant after argparse has parsed it? Using `sys` seems to beat the purpose of `argparse`. What do you think? Maybe calling a function to reset a set of arguments such as `*_TIMEOUT` after argparse has finished parsing CLI arguments? Maybe another better way?

Reply:
Yeah, agree with Tayfun here - we probably also don't want to set constants dynamically like this.

Reply:
@usc-m We have request timeout as CLI parameter which if provided should overwrite this default. This way, anywhere else in code which imports this constant will use CLI updated or default value. If we leave the constant as it is here, we will have to pass the CLI variable to any code which may use it and that code should use either that or this one which seems a little cumbersome?

Reply:
@joejuzl Do you have an idea?

Reply:
I'd say that fact that we're changing it dynamically shows it's not a constant (only the default value is a constant). So while it's cumbersome I feel like we should pass it from the CLI to wherever it's needed. This also means that we are not necessarily restricted to one global timeout, and can potentially handle it differently for different use cases (e.g. remote actions, model pulling etc.)